backend/core/models/user.py

```diff
@@ -35,8 +35,14 @@ class User(AbstractUser):
         max_length=100, blank=True, null=True, help_text="Last name"
     )
     title = models.TextField(null=True, blank=True, help_text="Your position")
+    diplomas = models.TextField(
+        null=True, blank=True, help_text="Your diplomas"
+    )
+    competencies = models.TextField(
+        null=True, blank=True, help_text="Your competencies"
+    )
```

Comment:
Hello @amatissart, thank you a lot for all those reviews!
Here is the idea I have about vouching mechanism, which answers some of your questions below : 
I want users to be able to tell how much they trust other users on different aspects (I can vouch for someone because I know they are who they pretend to be, or because I know they have the diplomas or competencies they claim). I think we can remove fields of diplomas and competencies for now if we think it is sufficient for users to say how much they vouch for a profile globally, which gives users more freedom on how they assess the importance of the different aspects they can vouch for by giving them a trust value. I was thinking that it would be easier to let users give trust scores to users without constraint (other than the trust value being non-negative, I'll enforce that thank you for the remark!) so that they can choose their metrics more easily, and then scale the result within the algorithm (the algorithm is implemented, I will do a PR for it once it is cleaned and documented so that you can see how it works). Do you think it would be a better option to enforce a certain metric in terms of user experience? 
Regarding the attribute, it was for users to be able to select what they were vouching for about (the PoP, diplomas or competencies mostly) but if we remove them I think we can also remove the attribute field too, at least for now?

Reply:
> Do you think it would be a better option to enforce a certain metric in terms of user experience?

The UX will need to be discussed with the team. But if this "trust_value" is given by the user themselves, then I think an explicit scale would be preferable. Otherwise low values could be ambiguous. 

> Regarding the attribute, it was for users to be able to select what they were vouching for about (the PoP, diplomas or competencies mostly) but if we remove them I think we can also remove the attribute field too, at least for now? 

As you wish. If you think we can do without this `attribute`, then it's okay with me.